spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/HttpClientConfigurableHttpConnectionFactory.java

```diff
@@ -99,7 +99,7 @@ private HttpClientBuilder lookupHttpClientBuilder(final URL url) {
 
 		if (builderMap.isEmpty()) {
 			this.log.warn(String.format("No custom http config found for URL: %s", url));
-			return HttpClients.custom();
```

Comment:
Why this change?

Reply:
I changed the code to give jGit the oppertinity to use their own HttpClient for standard cases. It is not necessary but prevents incompatible configuration of HttpClient between jGit and SpringCloudConfig

Reply:
I can drop that change (and the related changes to the unit test).

Reply:
Good idea, but

- using HttpClientSupport needs either JGitEnvironmentProperties or MultipleJGitEnvironmentProperties and
- has to be handling GeneralSecurityException

IMO the effort is too much in comparison to letting jGit handle the creation of the HttpClient

Reply:
OK please ad a comment that we are letting JGit handle it